### PR TITLE
fix(oauth): fix scope validation on empty scopes

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: app/scripts/models/reliers/oauth.js [hunk 1]
```diff
@@ -330,7 +330,7 @@ function replaceItemInArray(array, itemToReplace, replaceWith) {
 }
 
 function scopeStrToArray(scopes) {
-  if (! _.isString) {
```
Comment (shane-tomlinson) [inline app/scripts/models/reliers/oauth.js]: Wow....

Comment (shane-tomlinson) [reply app/scripts/models/reliers/oauth.js]: Can you add a test for this?

Comment (vladikoff) [reply app/scripts/models/reliers/oauth.js]: I'm also moving this function in the pairing PR into the base relier to make it testable and accessible. gotta figure out how to manage that...

**Comment by vladikoff** [comment]
@shane-tomlinson updated, and this also for https://github.com/mozilla/fxa-content-server/pull/6876/files#diff-ca4d7096469c2a76694bb8214760cbaaR115 now

**Pull request merged**